examples/connect/server.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+// eslint-disable-next-line
+const tracing = require('./tracing')('example-connect-server');
+
+// Require in rest of modules
+const connect = require('connect');
+const axios = require('axios');
+
+// Setup express
+const app = connect();
+const PORT = 8080;
+
+app.use(function middleware1(req, res, next) {
+  next();
+});
+
+app.use((req, res, next) => {
+  next();
+});
+
+app.use('/run_test', async (req, res) => {
+  const result = await axios.get('https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/main/package.json');
+  tracing.log('sending response');
```

Comment:
I'd just use `console.log`.

Reply:
it has additional time so it is easier to compare it with time of span